### ISSUE TITLE
Error: missing field `leave` when logging in

### ISSUE THREAD

**Issue by strct** [DevinR528/rumatui#6]
Error when a room was added in another session and logging in again. (Server: [conduit](https://conduit.rs))

```
An error occurred while serializing or deserializing.
missing field `leave` at line 1 column 11262
```

Bumping ruma version to after https://github.com/ruma/ruma/commit/acff664671e3f53bd012d33228363780eb20cf35 could fix the issue.

Since it adds https://github.com/ruma/ruma/commit/acff664671e3f53bd012d33228363780eb20cf35#diff-ab070b8339161ade1299b909d63e50a3R133

**Comment by DevinR528** [comment]
I'll release `0.1.18` as soon as the tests pass for #8 so you can just `cargo install --git https://github.com/DevinR528/rumatui` that has the very latest `ruma` because `matrix-rust-sdk` has also been updated (I'm tied to the version of ruma the SDK uses).

Again thank you for the issue!!

**Comment by strct** [comment]
Fixed by #8
